### ISSUE TITLE
Teacher model performance 

### ISSUE THREAD

**Issue by mumichang** [chengshengchan/model_compression#3]
Hi. Thanks for sharing your code. I am wondering why your teacher's model (NIN) has 78% accuracy, which is lower than the one reported in the paper.

**Comment by chengshengchan** [comment]
Thanks for your question.
In this work, the teacher network (network in network) was trained from scratch by myself.
Maybe there some flaws during the training procedure. 
Since the main purpose of this work is verifying the distilling ability.
I simply used default hyperparameters in caffe, and didn't try to optimize the performance.
